docs/guides/lint-arazzo.md

```diff
@@ -120,6 +120,16 @@ To expand the linting checks for an Arazzo description, start by enabling
 some of the built-in rules. The currently-supported rules are:
 
 - `parameters-no-body-inside-in`: the `in` section inside `parameters` must not contain a `body`.
+- `strict-source-description-type`: the `type` property of the `sourceDescription` object must be either `openapi` or `arazzo`.
+- `arazzo-version-enum`: the `version` property must be one of the allowed values: [`1.0.0`].
+- `workflow-workflowId-unique`: the `workflowId` property must be unique across all workflows.
+- `step-stepId-unique`: the `stepId` must be unique amongst all steps described in the workflow.
+- `sourceDescription-name-unique`: the `name` property of the `sourceDescription` object must be unique across all source descriptions.
```

Comment:
The spec rule will display error - the sourceDescription name must be unique according to the spec.

Reply:
I don't understand why I need this rule as well if it's in spec?

Reply:
Otherwise you won't get messages when you make a mistake - and will notice the problem only during the run.
Its just help you to identify problem earlier.